### ISSUE TITLE
Inference for a single image

### ISSUE THREAD

**Issue by gyan2976** [large-ocr-model/large-ocr-model.github.io#12]
How can I perform text recognition for a single image? You have used lmdb format to test the data but how to test a image without any format?

**Comment by ZhonghaoLu** [comment]
> How can I perform text recognition for a single image? You have used lmdb format to test the data but how to test a image without any format?

Have you solved it? I'm also facing the same problem

**Comment by gyan2976** [comment]
> > How can I perform text recognition for a single image? You have used lmdb format to test the data but how to test a image without any format?
> 
> Have you solved it? I'm also facing the same problem

Yes, basically this is [CLIP4STR](https://github.com/VamosC/CLIP4STR) a base model. Here in inference section you can find the command for inference. You just need to use your large-ocr-model weight.

For example
bash scripts/read.sh 0 clip4str_base16x16_d70bde1f2d.ckpt misc/test_images

Output:
image_1576.jpeg: Chicken
